cfme/tests/control/test_alerts_misc.py

```diff
@@ -0,0 +1,71 @@
+import pytest
+import fauxfactory
+from utils.update import update
+from cfme.control import explorer
+import cfme.fixtures.pytest_selenium as sel
+
+
+@pytest.mark.meta(blockers=[1303645], automates=[1303645])
+def test_control_alert_copy(soft_assert):
+    alert = explorer.Alert(
+        fauxfactory.gen_alphanumeric(), timeline_event=True, driving_event="Hourly Timer"
+    )
+    alert.create()
+    soft_assert(alert.exists, "The alert {} does not exist!".format(
+        alert.description
+    ))
+    alert_copy = alert.copy()
+    soft_assert(alert_copy.exists, "The alert {} does not exist!".format(
+        alert_copy.description
+    ))
+    alert_copy.delete()
+    soft_assert(not alert_copy.exists, "The alert {} exists!".format(
+        alert_copy.description
+    ))
+    alert.delete()
+    soft_assert(not alert.exists, "The alert {} exists!".format(
+        alert.description
+    ))
+
+
+def test_alert_profile_crud(soft_assert):
+    alert = explorer.Alert(
+        fauxfactory.gen_alphanumeric(),
+        timeline_event=True,
+        driving_event="Hourly Timer"
+    )
+    alert.create()
+    alert_profile = explorer.VMInstanceAlertProfile(
+        fauxfactory.gen_alphanumeric(),
+        [alert]
+    )
+    alert_profile.create()
+    soft_assert(alert_profile.exists, "The alert profile {} does not exist!".format(
```

Comment:
We spoke about `assert` X `soft_assert` and this would be another place to think through the scenarios and use the correct assert. Does it make sense to proceed with the test by trying to update and delete here if we weren't able to create the policy in the first place? Probably not - it would only give us false failures. :wink: